services/cassandra.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash -e
 # Begin service ENV variables
-export SHIPPABLE_CASSANDRA_PORT=9042;
-export SHIPPABLE_CASSANDRA_BINARY="/usr/sbin/cassandra";
-export SHIPPABLE_CASSANDRA_CMD="$SHIPPABLE_CASSANDRA_BINARY -R";
-export SHIPPABLE_CASSANDRA_LOG="/var/log/cassandra/system.log"
+service_name="cassandra"
+service_services_dir="$(dirname $(which shippable_service))/shippable_services"
+service_env_script="$service_services_dir/$service_name"_env.sh
```

Comment:
I don't think this needs to be so complicated. The env file can be assumed to be in the same directory as the service script. `dirname "$0"` should be sufficient.